src/patternfly/layouts/Page/page.hbs

```diff
@@ -1,17 +1,28 @@
-<div class="pf-l-page {{ modifierClass }}">
+<div class="pf-l-page{{#if pf-l-page--IsExpanded}} pf-m-expanded{{/if}} {{#if pf-l-page--modifiers}} {{pf-l-page--modifiers}}{{/if}}"
+  {{#if pf-l-page--attributes}} {{{pf-l-page--attributes}}} {{/if}}>
   <header role="banner" class="pf-l-page__header">
-    <section class="pf-l-page__header-sidebar">
-      Left
-    </section>
-    <section class="pf-l-page__header-main">
-      Right
-    </section>
+    <div class="pf-l-page__header-brand">
+      pf-c-Brand
+    </div>
+
+    <div class="pf-l-page__header-selector">
+      pf-c-context-selector
+    </div>
+
+    <div class="pf-l-page__header-nav">
+      pf-c-horizontal-nav
+    </div>
```

Comment:
Should this be `<nav>` instead? Also, I think design should standardize on the labels we apply to navigation, regarding what the label should be for the primary `<nav>` element, and what it should be for the secondary `<nav>` element if included.

I think we should always include `aria-label` with the `<nav>` elements in our examples. It's more of a usability issue if there are more than one `<nav>` elements, but it doesn't hurt to have a single `<nav>` labeled.

Reply:
I don't think so, even though this is the nav area it can contain other elements. Plus the nav component will have the nav tag